lsbi/stats.py

```diff
@@ -43,6 +45,33 @@ def _bar(self, indices):
         k[indices] = False
         return k
 
+    def bijector(self, x, inverse=False):
+        """Bijector between U([0, 1])^d and the distribution.
+
+        - x in [0, 1]^d is the hypercube space.
+        - theta in R^d is the physical space.
```

Comment:
Perhaps this docstring could clarify as this method is valid on likelihood or posterior/prior what physical space is. It feels most natural to define this for parameter space (theta doubly suggesting this) transformations or some comment on it's dual usage (if it is intended/makes sense to use on data distributions too)